### ISSUE TITLE
The header user menu requires two taps to open

### ISSUE THREAD

**Issue by AlexandraMoga** [mozilla/addons-frontend#11137]
Follow up for #11135 

### Describe the problem and steps to reproduce it:
1. Load AMO -dev on an android device
2. Log in
3. Tap on your user name in the page header to open the user menu

### What happened?
The user menu does not open at the the first tap, only at the second one

### What did you expect to happen?
The user menu opens at the first tap

![studio64_Nfzt3vO7JK](https://user-images.githubusercontent.com/31961530/153563750-b1561105-0ab6-40a8-bc06-a6f3fed22bae.gif)

**Comment by willdurand** [comment]
looks _very_ similar to https://github.com/mozilla/addons-frontend/issues/10515, iirc that's the same component under the hood

**Comment by AlexandraMoga** [comment]
> looks _very_ similar to #10515, iirc that's the same component under the hood

Yes, they do. The only difference is that the More menu is hidden on Android by default, but the user menu is always displayed, which raises the priority of this issue I'd say.

**Comment by diox** [comment]
I can unfortunately reproduce. Like you, I see this on dev and not on stage, indicating it has probably regressed with the removal of fastclick for some reason. I'll check locally later to be sure...

**Comment by AlexandraMoga** [comment]
The header user menu is opening at he first tap now.

This issue is verified fixed on stage with Android 11 and Android 10.
